theano/sandbox/blocksparse.py

```diff
@@ -2,7 +2,6 @@
 
 import theano
 from theano import Op, Apply
-from theano import tensor
```

Comment:
@bouthilx Having this line makes the code crash, I don't really understand why, probably an import cycle?